src/helpers/areCardsEqual.js

```diff
@@ -0,0 +1,9 @@
+export default (card1, card2) => {
+  if (!card1) {
+    return !card2
+  }
+  if (!card2) {
+    return false
+  }
+  return card1.id === card2.id && card1.idx === card2.idx
```

Comment:
I think something like this might be a little more permissive:

```js
const areCardsEqual = (a, b) => {
  if (!a.id || !b.id) return false
  if (!a.idx || !b.idx) return a.id === b.id
  return a.id === b.id && a.idx === b.idx
}
```